lib/validators/checklist_criteria_validator.rb

```diff
@@ -4,7 +4,7 @@ module Validators
   class ChecklistCriteriaValidator < QrdaFileValidator
     include Validators::ChecklistResultExtractor
     include Validators::Validator
-    include HealthDataStandards::Validate
+    include ::CqmValidators
```

Comment:
If you don't need the `::` in `cms_schematron_validator.rb` (below), can you get rid of it here? Not sure if that's possible, but it'd be cleaner if so. That, or add it in `cms_schematron_validator.rb` for consistency.